lib/active_merchant/billing/gateways/priority.rb

```diff
@@ -161,21 +161,22 @@ def exp_date(credit_card)
       end
 
       def add_type_merchant_purchase(params, merchant, is_settle_funds, options)
-        params['cardPresent'] = false
-        params['cardPresentType'] = 'CardNotPresent'
-        params['isAuth'] = true
+        params['cardPresent'] = options[:card_present]? options[:card_present] : 'false'
```

Comment:
Does Ruby ternary work with 

`options[:card_present]? options[:card_present] : 'false'`

vs

`options[:card_present] ? options[:card_present] : 'false'`  ?

I never knew that!

Reply:
It was typo fixed now.